app/javascript/app/acuant/sdk.js

```diff
@@ -0,0 +1,48 @@
+const {
+  fetchSdkInitializationCredentials,
+  fetchSdkInitializationEndpoint,
+  acuantSdkInitializationStarted,
+  acuantSdkInitializeSuccess,
+  acuantSdkInitializeFailed,
+  addClickEventListenerToAcuantCaptureButton,
+  acuantImageCaptureStarted,
+  acuantImageCaptureSuccess,
+  acuantImageCaptureFailed,
+} = require('./domUpdateCallbacks');
+
+export const imageCaptureButtonClicked = (event) => {
+  event.preventDefault();
+  acuantImageCaptureStarted();
+  window.AcuantCameraUI.start(
+    acuantImageCaptureSuccess,
+    acuantImageCaptureFailed,
+  );
+};
+
+export const initializeAcuantSdk = (credentials = null, endpoint = null) => {
+  credentials = credentials || fetchSdkInitializationCredentials();
+  endpoint = endpoint || fetchSdkInitializationEndpoint();
+  window.AcuantJavascriptWebSdk.initialize(
+    // Dummy credentials for Acuant frontend
```

Comment:
now this comment is a lie

Reply:
Well, they're still dummy credentials. I really want to make it clear that these are not secret and I meant to commit them.

Reply:
Maybe that comment should got in `application.yml.default`? My concern there is that they'll go away if anyone ever normalizes that file.

Reply:
just put them at the end of the line instead of above?

```
acuant_sdk_initialization_creds: 'aaa' # dummy credential
```

or maybe make them dumb inside the secret

```
acuant_sdk_initialization_creds: 'SOME_FAKE_VALUE'
```